sdk/include/opentelemetry/sdk/configuration/span_limits_configuration.h

```diff
@@ -0,0 +1,29 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace configuration
+{
+
+// YAML-SCHEMA: schema/tracer_provider.json
+// YAML-NODE: SpanLimits
+class SpanLimitsConfiguration
+{
+public:
+  size_t attribute_value_length_limit;
+  size_t attribute_count_limit;
+  size_t event_count_limit;
+  size_t link_count_limit;
+  size_t event_attribute_count_limit;
+  size_t link_attribute_count_limit;
```

Comment:
I am not sure if otel-cpp implementation supports all these configurations. Do we still need to add them?

Reply:
The goal is for the yaml parser to translate all the yaml syntax into C++ model data (the *_configuration classes), so I would like to have complete coverage there, to make sure the parser understand the full syntax.

Now, down the line, when it is time to instantiate a SDK object from the model, of course there will be some features not supported by the opentelemetry-cpp SDK, so that warnings or errors will be raised then.

That part of the code (create SDK objects from the model) is not in the scope for this PR, it will be done in PRs to come that consume the model.